scripts/xtensa-build-all.sh

```diff
@@ -6,7 +6,7 @@
 set -e
 
 SUPPORTED_PLATFORMS=(byt cht bdw hsw apl skl kbl cnl sue icl jsl \
-                    imx8 imx8x imx8m tgl tgl-h)
+                    imx8 imx8x imx8m tgl tgl-h rn)
 BUILD_ROM=no
```

Comment:
we should rebase this as we added support for imx8ulp.

Reply:
Addressed the Review Comment. Done